lib/animation.dart

```diff
@@ -55,6 +55,7 @@ class Animation {
   ///
   /// From a single image source, it creates multiple sprites based on the parameters:
   /// [amount]: how many sprites this animation is composed of
+  /// [amountPerRow]: how many sprites per row
```

Comment:
I like this feature, but since it's a bit more complicated to understand, can we have a bit more text on the docs explaining what this means? maybe with an example...